### PR TITLE
Decouple policies and validator/kernel

### PR DESCRIPTION
**Pull request by JohnPMerrill**
See draperlaboratory/hope-src#52

### CODE REVIEW

File: validator/include/sim_validator.h [hunk 1]
```diff
@@ -28,26 +28,24 @@
 #define RENODE_VALIDATOR_H
```
Comment (amstrnad) [inline validator/include/sim_validator.h]: Should this also be renamed?

Comment (JohnPMerrill) [reply validator/include/sim_validator.h]: Got it, thanks for the catch

File: validator/include/sim_validator.h [hunk 2]
```diff
@@ -24,30 +24,28 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#ifndef RENODE_VALIDATOR_H
-#define RENODE_VALIDATOR_H
+#ifndef SIM_VALIDATOR_H
+#define SIM_VALIDATOR_H
 
 #include "validator.h"
-#include "renode_interface.h"
```
Comment (ccasin) [inline validator/include/sim_validator.h]: Can `policy-engine/validator/include/renode_interface.h` be deleted as a result of this change?

Comment (JohnPMerrill) [reply validator/include/sim_validator.h]: done

**Pull request merged**